### PR TITLE
Add swoole

### PR DESCRIPTION
**Pull request by hhxsv5**
[Swoole](https://github.com/swoole/swoole-src) is the Coroutine-based concurrency library for PHP.

### CODE REVIEW

File: php/swoole/config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+framework:
+  github: swoole/swoole-src
+  version: 4.6
+
+php_ext:
+  - swoole
+
+command: php bin/laravels start
```
Comment (waghanza) [inline php/swoole/config.yaml:8]: ?

Comment (hhxsv5) [reply php/swoole/config.yaml]: Sorry for that, modified

**Comment by waghanza** [comment]
Seems to be related to https://github.com/the-benchmarker/web-frameworks/discussions/3789

**Comment by sy-records** [comment]
@waghanza Is the intention to add Swoole? If so, I have time to add a new one, as Swoole has multiple implementations

**Comment by waghanza** [comment]
> @waghanza Is the intention to add Swoole?

yes. this break up my plans, but seems to be an amazing feature, hop other communities will join on this effort

File: php/swoole/composer.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "require": {
+  }
+}
```
Comment (waghanza) [inline php/swoole/composer.json:4]: I suggest having the following manifest
```suggestion
{
  "require": {
    "ext-swoole": "~4.6.0",
  }
}
```
it ensure that we are in-sync with version in the doc

File: php/swoole/start.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Swoole\Http\Server;
+use Swoole\Http\Request;
+use Swoole\Http\Response;
+
+$server = new Server('0.0.0.0', 3000);
+$server->set([
+    'daemonize'  => false,
+    'worker_num' => swoole_cpu_num() * 2,
```
Comment (waghanza) [inline php/swoole/start.php:10]: please use the same config as existing (made by @sy-records), I need to find an easiest way to include them
```suggestion
   'worker_num' => swoole_cpu_num() * 2,
    'enable_coroutine' => false,
    'log_file' => '/dev/null',
    'log_level' => SWOOLE_LOG_ERROR,
```

**Review by waghanza — CHANGES_REQUESTED** [review]
minor changes, but thanks for this

**Comment by hhxsv5** [comment]
Modified, please review.

File: php/swoole/start.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Swoole\Http\Server;
+use Swoole\Http\Request;
+use Swoole\Http\Response;
+
+$server = new Server('0.0.0.0', 3000);
+$server->set([
+    'worker_num'       => swoole_cpu_num() * 2,
+    'pid_file'         => storage_path('laravels.pid'),
```
Comment (sy-records) [inline php/swoole/start.php:10]: Useless code

**Review by sy-records — CHANGES_REQUESTED** [review]

File: php/swoole/start.php (hunk 4 above)
Comment (hhxsv5) [reply php/swoole/start.php]: Fixed

**Review by waghanza — APPROVED** [review]

**Pull request merged**